frontend/src/reports/ui/IncarceratedChildrenLongAlert.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Alert from "@material-ui/lab/Alert";
+import AlertTitle from "@material-ui/lab/AlertTitle";
+import styles from "./IncarceratedChildrenLongAlert.module.scss";
+import { Link } from "react-router-dom";
+import FlagIcon from "@material-ui/icons/Flag";
+import { METHODOLOGY_TAB_LINK } from "../../utils/internalRoutes";
+
+function IncarceratedChildrenLongAlert() {
+  return (
+    <div>
+      <Alert
+        severity="error"
+        className={styles.ReportAlert}
+        icon={<FlagIcon />}
+        role="note"
+      >
+        <AlertTitle>Children in Adult Prisons</AlertTitle>
+
+        <p>
+          Although the criminal justice system makes distinctions between
+          children and adults, there are state-level laws that remove children
+          from the protective cover of these distinctions and enable the
+          incarceration of children in adult correctional institutions.
+        </p>
+        <p>
+          When showing prison reports filtered by age, we highlight the{" "}
+          <b>total number of confined children</b>, rather than a ‘per 100k’
+          rate calculated using an ambiguous population base. Read more in{" "}
```

Comment:
Hmm I agree. I wanted to explain the fact that we don't really know what the population number is to divide by, since it's not REALLY ages 0-17 because children under 15 almost never are in prison... but it's slightly different for each jurisdiction and offense, etc. So maybe I'll leave off the `, rather than a ‘per 100k’ rate calculated using an ambiguous population base. ` part and explain that in more depth in the methodology